pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -2060,6 +2060,12 @@ class Timedelta(_Timedelta):
             # integers or floats
             if util.is_nan(other):
                 return NaT
+            # We want NumPy numeric scalars to behave like Python scalars
+            # post NEP 50
+            if isinstance(other, np.integer):
```

Comment:
can we use cnp.integer/cnp.floating for these?  i think somewhat more performant

Reply:
updated